tests/acceptance-tests/test_nested_mir.cpp

```diff
@@ -52,6 +52,7 @@
 #include "mir/test/doubles/stub_cursor.h"
 #include "mir/test/doubles/stub_display_configuration.h"
 #include "mir/test/signal_actions.h"
+#include "../include/mir/test/doubles/mock_display_configuration_observer.h"
```

Comment:
That should be #include "mir/test/doubles/mock_display_configuration_observer.h"

Reply:
oops

Reply:
Actually that doesn't work here, and I'm not sure why